schematic/__init__.py

```diff
@@ -27,10 +27,14 @@
 from schematic.loader import LOADER
 from schematic.version import __version__
 from schematic_api.api.security_controller import info_from_bearer_auth
+from dotenv import load_dotenv
```

Comment:
@linglp does this need to be added to the poetry.lock file?

Reply:
we already have `from dotenv import load_dotenv` in conftest, so I don't think we need to add it to the lock file.

Reply:
`conftest` is tests. This is now part of `__init__.py` which is part of the API and CLI loading. if `dotenv` is not installed in the general `pip install schematicpy` then this will cause problems

Reply:
https://github.com/Sage-Bionetworks/schematic/blob/aa851e3c04689e9574913a1340f923b153c20c8c/pyproject.toml#L91

That's right! It would need to be moved to the regular project dependencies in order to install with the package.

Reply:
@linglp what happens when the .dotenv file doesn't exist?  My concern is failing AWS infra deployment

Reply:
@thomasyu888 thanks for the question, Tom! I read the source code of `dotenv` here: https://github.com/maxkoryukov/python-dotenv/blob/311f56cf524a9e2b38a65f210170e4e631778b35/dotenv/main.py#L19 and it seems like if .env does not exist, `load_dotenv` will just return None.

Reply:
@linglp , this is helpful.

Can you add to the readme that there are two ways to provide environmental variables when using schematic CLI? 

1. `export ...` and/or add to `~/.bashprofile`
2. Create .env file and add the the env variables required.

Given that the schematic API deployment won't ever use `.env` method, I think we don't have to include those instructions in the API deployment docs.

Reply:
Thank you, Tom! I have added the instructions.